.github/workflows/cypress-workflow.yml

```diff
@@ -0,0 +1,129 @@
+          echo "::set-output name=yarn_version::$(node -p "(require('./kibana/package.json').engines.yarn).match(/[.0-9]+/)[0]")"
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ steps.versions.outputs.node_version }}
+          registry-url: 'https://registry.npmjs.org'
+      - name: Install correct yarn version for Kibana
+        run: |
+          npm uninstall -g yarn
+          echo "Installing yarn ${{ steps.versions_step.outputs.yarn_version }}"
+          npm i -g yarn@${{ steps.versions.outputs.yarn_version }}
+      - name: Bootstrap plugin/kibana
+        env:
+          CYPRESS_INSTALL_BINARY: ${{ steps.cypress_version.outputs.cypress_version }}
+        run: |
+          mkdir -p kibana/plugins
+          mv security-kibana-plugin kibana/plugins
+          cd kibana/plugins/security-kibana-plugin
+          yarn kbn bootstrap
+      - name: Run kibana server
+        run: |
+          security_config='elasticsearch.username: "kibanaserver"
+          elasticsearch.password: "kibanaserver"
+          elasticsearch.hosts: "https://localhost:9200"
+          elasticsearch.ssl.verificationMode: none
+          elasticsearch.requestHeadersWhitelist: ["securitytenant","Authorization"]
+          opendistro_security.multitenancy.enabled: true
+          opendistro_security.multitenancy.tenants.preferred: ["Private", "Global"]
+          opendistro_security.readonly_mode.roles: ["kibana_read_only"]
+          opendistro_security.auth.type: "basicauth"
+          '
+          echo "$security_config" >> kibana/config/kibana.yml
+          cat kibana/config/kibana.yml
+          cd kibana
+          yarn start --no-base-path --no-watch &
+          sleep 300
+          curl localhost:5601/api/status
+      - name: Cache Cypress
+        id: cache-cypress
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/Cypress
+          key: cypress-cache-v2-${{ runner.os }}-${{ hashFiles('**/package.json') }}
+      # now let's install Cypress binary
+      - run: npx cypress install
+      - run: npx cypress cache list
+      - run: npx cypress cache path
+        # for now just chrome, use matrix to do all browsers later
+      - name: Cypress tests
+        run: |
+          cd kibana/plugins/security-kibana-plugin
+          yarn run cypress run --browser chrome
+      # Screenshots are only captured on failure, will change this once we do visual regression tests
+      - uses: actions/upload-artifact@v1
+        if: failure()
+        with:
+          name: cypress-screenshots
+          path: kibana/plugins/security-kibana-plugin/cypress/screenshots
+      # Test run video was always captured, so this action uses "always()" condition
+      - uses: actions/upload-artifact@v1
+        if: always()
+        with:
+          name: cypress-videos
+          path: kibana/plugins/security-kibana-plugin/cypress/videos
```

Comment:
should we also upload the build artifacts ? Very helpful to just download and get run locally.

Reply:
Can you please elaborate more on this? If you are asking how to run the integ test locally, currently we need to run an ES separately and then yarn run cypress open

Reply:
I meant to say here that, can we download the just built plugin zip file ?

Reply:
There is a CI workflow who is for building and uploading plugin zip.
https://github.com/opendistro-for-elasticsearch/security-kibana-plugin/blob/master/.github/workflows/ci.yml

If this is what you were talking about, I think it has been done by the ci workflow.